code/game/objects/structures/statues.dm

```diff
@@ -350,3 +350,14 @@
 	name = "Statue of an assistant"
 	desc = "A cheap statue of sandstone for a greyshirt."
 	icon_state = "assist"
+
+/////////////////////snow/////////////////////////////////////////
+
+/obj/structure/statue/snow
+	hardness = 0.5
+	mineralType = "sandstone"
```

Comment:
jokes on you the entire mission is actually just white sand

in other words copypasta is bad